src/main/reseal.c

```diff
@@ -135,6 +135,7 @@ static void usage(const char *prog)
           " -f or --force           Force the overwrite of an existing .ski file when using default output.\n"
           " -p or --pcrs_list       List of TPM platform configuration registers (PCRs) to apply to authorization policy.\n"
           "                         Defaults to no PCRs specified. Encapsulate in quotes (e.g. \"0, 1, 2\").\n"
+          " -g                      Used when resealing a file that already contains a policy OR\n" // may change later
```

Comment:
So, my original thought was to initially assume (i.e. require) that the input file to be 'resealed' would not have a policy-OR authorization criteria.

The real fix to this (i.e., the need for a command-line option like this) is the update that addresses Issue #173. When code to address that issue is merged, the detection of a policy-OR criteria from the .ski file format will be automatic and no longer require a command-line option.

I think inclusion of this option, for now, although kind of clunky, makes `kmyth-reseal` more generic, which is a good thing.